gym_ignition/envs/base/ignition_python_env.py

```diff
@@ -37,40 +34,47 @@ def __init__(self):
         self._gazebo_wrapper = None
         self._model_sdf = None
         self._world_sdf = None
-        self._iterations = 1
-        self._update_rate = 100.0
+        self._iterations = None
+        self._controller_dt = None
+        self._physics_rate = 1000000000.0
```

Comment:
I am a bit confused by this. I would expect the `physics_rate` to be the inverse of the physics integrator time step, but I don't think we are currently using `1 ns` as our physics timestep.

Reply:
It's unclear to be honest how to set the integrator time step of the simulator. The exposed APIs of ignition gazebo allow set the update rate of the simulation, and it is not clear if it is associated to the physics integration or if within a single simulator iterations the backend can decide to step the physics more than once.

Reply:
As far as I could quickly see on the current default branch of `ign-gazebo`, as far as a single `Step` in the ignition-physics plugin correspond to a single step in the underlying fixed-time step integrator, indeed what you have there is  the physics update rate, i.e. the inverse of the physics update time step. For this reason, I am even more curious about this default value of 1 ns for the physics update. Does it make sense? From where does it come from?

Reply:
As discussed f2f the physics rate was descreased to `1000 Hz` by default, and the simulation tries to run as fast as possible setting the RT factor in the sdf file.